packages/e2e/package.json

```diff
@@ -22,7 +22,7 @@
     "@types/chai": "^4.2.0",
     "@types/fs-extra": "^5.1.0",
     "@types/mocha": "^5.2.6",
-    "@types/node": "^10.17.24",
+    "@types/node": "^16.11.9",
```

Comment:
We normally keep this aligned with the oldest version of node that we support, which is 12.x